.gitignore

```diff
@@ -4,4 +4,3 @@ npm-debug.log.*
 htmldocs/
 lerna-debug.log
 lib/
-.DS_Store
```

Comment:
One way of tackling it, but personally I don't like using global git settings for anything par user name/email—I believe it's much safer to have this kind of stuff configured on a project by project basis. That way if someone works on the project and don't have global settings that ignore these system files (like me) they don't end up accidentally committing these files. Your repo, your call—just my 2 cents :)